LibraBFT/Impl/Consensus/RoundManager/PropertyDefs.agda

```diff
@@ -22,18 +22,17 @@ module LibraBFT.Impl.Consensus.RoundManager.PropertyDefs where
 -- source meta-data. The actual statement of the property depends on the
 -- stateful computation we are considering (e.g., whether the vote is being
 -- returned or emitted as output).
-VoteSrcCorrectCod : (pre post : RoundManager) → VoteWithMeta → Set
-VoteSrcCorrectCod pre post (VoteWithMeta∙new vote mvsNew) =
-  just vote ≡ post ^∙ lSafetyData ∙ sdLastVote
-VoteSrcCorrectCod pre post (VoteWithMeta∙new vote mvsLastVote) =
-  just vote ≡ pre ^∙ lSafetyData ∙ sdLastVote
-  × pre ≡L post at (lSafetyData ∙ sdLastVote)
+
+data VoteSrcCorrectCod (pre post : RoundManager) (vote : Vote) : Set where
```

Comment:
Overall this seems much better than cluttering up the implementation with meta-data!